documentation/ab-testing-with-google-optimize.md

```diff
@@ -0,0 +1,29 @@
+# A/B testing with Google Optimize
+
+## Setting up and a new redirect experiment
+- On the [Google Optimize console](https://optimize.google.com/) under the account,
+select a container for the environment you wish to run an experiment on
+
+- Create a new `Redirect Test` - this will allow to testing two different landing pages, with different URLs. 
+
+- Add the URL variants e.g `?variant=1` and `?variant=2`
```

Comment:
Maybe mention that you also need the application to behave differently depending on the value of the parameter passed in, and also reference the "button placement" PR as an example?

Reply:
Resolved this in be57ef3